content/reference/current/user/simulations/index.md

```diff
@@ -95,7 +95,17 @@ This jar must have been built with the same maven/sbt/gradle configuration as de
 
 {{< img src="create-simulation2c.png" alt="Create simulation - Step 2 - Binary" >}}
 
-- **Artifact Maven coordinates**: the maven coordinates of the desired artifact. _version markers are not supported_
+- **Artifact Maven coordinates**: the maven coordinates of the desired artifact.
+
+The following version markers can be used:
+
+* `latest.integration`: latest version with snapshots included
+* `latest.release`: same with snapshots excluded
+* `latest.stable`: same but excluding versions that seem to have a commit number in them
```

Comment:
I'm actually wondering if that one is useful to document...

Reply:
Agree. From the comments I read, it seems to be very approximative and opinionated. Please remove this last one.